amazon_config.template_json

```diff
@@ -2,5 +2,6 @@
   "username": "",
   "password": "",
   "asin_list": ["B07JH53M4T","B08HR7SV3M"],
-  "amazon_website": "amazon.co.uk"
+  "amazon_website": "amazon.co.uk",
```

Comment:
add "www." or "smile.". I don't think "smile.amazon.*" exists for all language versions of the amazon site.